src/Npgsql/NpgsqlLargeObjectManager.cs

```diff
@@ -113,7 +113,8 @@ public NpgsqlLargeObjectStream OpenRead(uint oid)
         /// <returns>An NpgsqlLargeObjectStream</returns>
         public Task<NpgsqlLargeObjectStream> OpenReadAsync(uint oid, CancellationToken cancellationToken)
```

Comment:
The same question I want to ask here and below.

Reply:
Here the situation is different, as it's not a standard ADO.NET API, and there aren't overloads without a cancellation token. I will go over our async surface API and add default on cancellation tokens where needed.

Reply:
(totally unrelated - I wonder if anyone is actually using the large object API :) I've never heard from any user about it...)